### ISSUE TITLE
opacity in Textstyles not working

### ISSUE THREAD

**Issue by Mauns** [ingrammicro/puzzle-tokens#35]
Hey, the opacity property for Textstyles doesn't seem to work …

Other than that great plugin!

**Comment by MaxBazarov** [comment]
Fixed in 8.14.2 
Enjoy!

**Comment by Mauns** [comment]
Cool, thanks for the quick reaction! But it doesn’t seem to work yet 😅 

**Value in Scss File**
```scss
@each $color-key, $text-color in $text-colors {
    .#{$color-key} {
        font-family: "Prelo";

        opacity: 100%; // ← OPACITY VALUE

        color: $text-color;

        text-align: $text-align;
        font-weight: $text-weight;

        font-size: map.get($map: $font-size, $key: "size");
        -pt-paragraph-spacing: map.get($map: $font-size, $key: "paragraph-spacing");
        line-height: map.get($map: $font-size, $key: "line-height");

    }
}
```
**Value in Sketch**
![Bildschirmfoto 2020-10-30 um 09 36 42](https://user-images.githubusercontent.com/10224551/97678006-6d3a8b00-1a93-11eb-8150-f1942a7ddc01.png)

**Comment by MaxBazarov** [comment]
It's my fault. I changed an alpha channel of color.
Ok, fixed again in 8.14.3 — now opacity: changes opacity!
BTW, it supports CSS format (0.0-1.0) too
